en/docs/guides/user-mgt/managing-user-attributes.md

```diff
@@ -0,0 +1,204 @@
+claim. Therefore, you need to use the claim management functionality
+available in WSO2 IS to properly map your LDAP/AD/JDBC user
+store attributes with the claim URIs defined by WSO2 IS. You
+can also add different claim URIs and manage them using claim
+management.
+
+The following topics provide instructions on how to manage user
+attributes in WSO2 IS.
+
+## Managing the attributes of a user
+
+The following are the three main ways to view, add, edit, and delete
+attributes of a user.
+
+1.  By accessing the profile of the user and changing the attributes
+    using the [admin Portal](link-admin-portal)
+2.  You can use the REST Web service according to the SCIM provisioning specification. For more information on this, see [Using the SCIM 2.0 REST APIs](link-using-the-scim-2.0-rest-apis).
+3.  You can use the `           RemoteUserStoreManagerService          `
+    API. This is a SOAP-based API and is very easy to use. For more
+    information on using this, see [Managing Users and Roles with
+    APIs](insert-link). Supposing you want to
+    set a user attribute, you can call the following method.
+
+    ``` java
+    setUserClaimValue("username", "http://wso2.org/claims/emailaddress", "mark@soasecurity.org", null)
+    ```
+
+    Here, `http://wso2.org/claims/emailaddress` is the claim URI that
+    has been mapped with the user store’s email attribute. The last
+    parameter is profile, we can just pass “null”, as there is only one
+    profile. You can retrieve the user attribute value as follows.
+
+    ``` java
+    getUserClaimValue("username", "http://wso2.org/claims/emailaddress", null)
+    ```
+
+## Claim mapping when using multiple user stores 
+
+{!fragments/xxx!}
+
+### Attributes with multiple values
+
+{!fragments/xxx!}
+
+### Writing custom attributes
+
+Supposing the attributes of a user are stored in both the user store
+(LDAP) and another location (JDBC table), WSO2 IS needs to
+retrieve/add the user’s attribute in both these places. In scenarios
+like this, you can simply extend the current user store manager implementation and write a
+custom implementation to do it. In the custom user store implementation,
+you only need to extend the following three methods that help to
+retrieve/add a user attribute. Other methods can be kept as they are.
+
+-   Method 1.
+
+    ``` java
+    public Map<String, String> getUserPropertyValues(String userName, String[] propertyNames, String profileName) throws UserStoreException
+    ```
+
+-   Method 2.
+
+    ``` java
+    protected abstract void doSetUserClaimValue(String userName, String claimURI, String claimValue, String profileName) throws UserStoreException;
+    ```
+
+-   Method 3.
+
+    ``` java
+    protected abstract void doSetUserClaimValues(String userName, Map<String, String> claims, String profileName) throws UserStoreException;
+    ```
+
+See [Writing a Custom User Store Manager](insert-writing-a-custom-user-store-manager) for more information on
+this.
```

Comment:
This content should go to the 'extend' section right? we can give a link from this page